Task 3/Task 3.1/Task 3.1/Task 3.1/Classes/Validator.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Task_3._1.Classes
+{
+    public static class Validator
+    {
+        /// <summary>
+        /// Method for user input
+        /// </summary>
+        /// <returns></returns>
+        public static int InputValue ()
+        {
+            string uservalue = Console.ReadLine();
+            int.TryParse(uservalue, out int value);
+            return value;
+        }
+
+        /// <summary>
+        /// Method for checking user input
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        public static int CheckValue(int value)
+        {
+            while (value <= 1)
+            {
+                Console.WriteLine($"Вы ввели неверное значение. Исправьте, пожалуйста.");
+                string uservalue = Console.ReadLine();
+                int.TryParse(uservalue, out value);
+            }
+            return value;
+        }
+
+
+        /// <summary>
+        /// Method for filling collection
+        /// </summary>
+        /// <param name="people"></param>
+        /// <param name="value"></param>
+        public static void FillList (PeopleList<int> people, int value)
+        {
+            int count = 1;
+            while (count <= value)
+            {
+                people.Add(count);
+                count++;
+            }
+            Console.WriteLine("Сгенерирован круг людей. Начинаем вычеркивать каждого второго.");
+        }
```

Comment:
It can create a collection and return it instead of working with already created one